wsbroadcastserver/clientconnection.go

```diff
@@ -56,7 +57,9 @@ func (cc *ClientConnection) Start(parentCtx context.Context) {
 			case data := <-cc.out:
 				err := cc.writeRaw(data)
 				if err != nil {
-					logError(err, "error writing data to client")
+					if !strings.Contains(err.Error(), "use of closed network connection") {
+						logWarn(err, "error writing data to client")
```

Comment:
Doesn't logWarn already have this if statement?

Reply:
:+1: